README.md

```diff
@@ -36,11 +36,11 @@ $ npm i @lumigo/tracer
 $ yarn add @lumigo/tracer
 ```
 
-* Next, wrap your `handler` in Lumigo's `trace` function (note: replace `YOUR-TOKEN-HERE` with your Lumigo API token):
+* Next, wrap your `handler` in Lumigo's `trace` function:
 
 ```javascript
 // javascript
-const lumigo = require('@lumigo/tracer')({ token: 'YOUR-TOKEN-HERE' })
```

Comment:
I'm pretty sure that users can set the token via an env var and they don't have to specify it here (They can if they want to).

I would direct users to set the env var and give this as an additional option

Reply:
That's what I've added in the `Connect Your Lumigo Account` section though right?

Reply:
Oh right my bad, thought you were adding the token to the code and not removing it.

1. Add the env var to the `Configuration` env vars list
2. I would explain in the `Configuration` entry for `LUMIGO_TRACER_TOKEN` env var that it can be specified directly in the code, in case users want to pass it in some other way
3. Also update the `Step Functions`, `Adding Execution Tags`  code segments

Reply:
Updated, WDYT?

Reply:
In the `Step Functions` I was talking about the part that you initialize the lumigo tracer with the token explicitly written in the js code